src/Tizen.NUI/src/public/RenderEffects/BackgroundBlurEffect.cs

```diff
@@ -0,0 +1,116 @@
+        /// <since_tizen> 13 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool BlurOnce
+        {
+            get
+            {
+                bool blurOnce = Interop.BackgroundBlurEffect.GetBlurOnce(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return blurOnce;
+            }
+
+            set
+            {
+                Interop.BackgroundBlurEffect.SetBlurOnce(SwigCPtr, value);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+        }
+
+        /// <summary>
+        /// The property is blur radius value.
+        /// The unit is pixel, but the property is in float type since many other platforms use float for blur effect radius.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float BlurRadius
+        {
+            get
+            {
+                uint blurRadius = Interop.BackgroundBlurEffect.GetBlurRadius(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return (float)blurRadius;
+            }
+
+            set
+            {
+                Interop.BackgroundBlurEffect.SetBlurRadius(SwigCPtr, (uint)Math.Round(value, 0));
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+        }
+
+        /// <summary>
+        /// Adds blur strength animation to the effect.
+        /// Basically it is to animate blurring clear texture, but when starting value(fromValue) is bigger than the end value(toValue), 
+        /// it may show a reversed animation that instead clarifies blurred texture.
+        /// </summary>
+        /// <param name="animation">Animation instance to add blur strength animation.</param>
+        /// <param name="alphaFunction">The alpha function to apply. If none, it will use animation's default alpha function.</param>
+        /// <param name="timePeriod">Duration of animation. If none, it will use the animation's duration.</param>
+        /// <param name="fromValue">Starting blur strength value of the animation. The value resides in range of [0,1].</param>
+        /// <param name="toValue">End of blur strength value of the animation. The value resides in range of [0,1].</param>
+        /// <since_tizen> 13 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AddBlurStrengthAnimation(Animation animation, AlphaFunction alphaFunction, TimePeriod timePeriod, float fromValue, float toValue)
+        {
+            Interop.BackgroundBlurEffect.AddBlurStrengthAnimation(SwigCPtr, animation.SwigCPtr, alphaFunction.SwigCPtr, timePeriod.SwigCPtr, fromValue, toValue);
```

Comment:
```suggestion
            Interop.BackgroundBlurEffect.AddBlurStrengthAnimation(SwigCPtr, Animation.getCPtr(animation), AlphaFunction.getCPtr(alphaFunction), TimePeriod.getCPtr(timePeriod), fromValue, toValue);
```

`~~~.getCPtr()` API allow to use null. Let we allow to null input for `alphaFunction` and `timePeriod`.